build.gradle

```diff
@@ -28,35 +53,14 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.0.0-SNAPSHOT'
-    compile group: 'org.opentest4j', name: 'opentest4j', version: '1.0.0-M1'
-
-    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.0.0-SNAPSHOT'
-    testCompile group: 'org.junit.platform', name: 'junit-platform-launcher', version: '1.0.0-SNAPSHOT'
+    api group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.0.0'
+    api group: 'org.opentest4j', name: 'opentest4j', version: '1.0.0'
 
-    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.5.2'
-    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.2.1'
-}
-
-buildscript {
-    repositories {
-        mavenLocal()
-        maven {
-            name "GradlePlugins"
-            url 'https://plugins.gradle.org/m2/'
-        }
-        maven {
-            name "MavenSnapshot"
-            url 'https://oss.sonatype.org/content/repositories/snapshots/'
-        }
-    }
+    testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.8.0'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.10.0'
 
-    dependencies {
-        classpath group: 'com.diffplug.gradle.spotless', name: 'spotless', version: '2.2.0'
-        classpath group: 'org.junit.platform', name: 'junit-platform-gradle-plugin', version: '1.0.0-SNAPSHOT'
-        classpath group: 'com.squareup.okhttp3', name: 'okhttp', version: '3.8.1'
-        classpath group: 'org.ajoberstar', name: 'grgit', version: '1.9.3'
-    }
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.0.0'
```

Comment:
Is it necessary to repeat `api` grouped dependencies as `testImplementation`? Looks redundant to me.

Reply:
Shouldn't have to repeat them, was a mistake by me. I will fix it.

Reply:
Just took another look at this:

`api` dependencies:

* `junit-jupiter-api
* `opentest4j`

Those make sense to me for limititing the transitive dependencies.

`testImplementation` dependencies

* `junit-jupiter-engine`
* `junit-platform-launcher`

Which I think are needed only in the tests because of usage of some pieces like https://github.com/junit-pioneer/junit-pioneer/tree/master/src/test/java/org/junit/platform/engine/test